src/test/java/DayTwoTest.java

```diff
@@ -19,6 +20,8 @@ public void givenMoreThanOneBiggerNumber_withNonFactorOfThree_CalculatesFuel(){
         Assertions.assertThat(dayTwo.getFuel("400")).isEqualTo(184);
 
     }
+
+    //naming?
```

Comment:
All these numbers seem to be the same size, I think the name isn't quite telling the truth.